### PR TITLE
fixed tooltip bug and closes #1679

### LINKED ISSUE #1679: 1.0.0-alpha1: tooltips: unexpected oder not shown
**Describe the bug**
tooltips are not displayed or show unexpected text

**To Reproduce**
Move over the icons

**Screenshots**
no tooltipps:

unexpected tooltip over share and action icon:
d1
 *  *  *  *  *
unexpected tooltip over avatar:
d2
 *  *  *  *  *
no: tooltips on add, filter, ...
d3

**Version**
deck: 1.0.0-alpha1
Nextcloud: 19.0.0 beta 2

### PR DESCRIPTION
**Pull request by ishantgupta777**
Signed-off-by: Ishant <ishantgupta777@gmail.com>

Fixed issue and resolves #1679  of the tooltip.

* Resolves: # 
* Target version: master

### CODE REVIEW

**Comment by juliushaertl** [comment]
Ideally the ActionsButton component from https://github.com/nextcloud/nextcloud-vue should just show a v-tooltip for single actions, but I'm fine with this as a temporary workaround.

File: src/components/boards/BoardItem.vue [hunk 1]
```diff
@@ -33,10 +33,11 @@
 			{{ board.title }}
 		</div>
 		<div class="board-list-avatars-cell">
-			<Avatar :user="board.owner.uid" class="board-list-avatar" />
+			<Avatar :user="board.owner.uid" :title="board.owner.displayname" class="board-list-avatar" />
```
Comment (juliushaertl) [inline src/components/boards/BoardItem.vue:36]: I think the tooltip here is coming from the parent element and the Avatar has a proper v-tooltip usage. My suggestion would be to just set an empty title attribute on the parent div (`.board-list-avatars-cell`)

File: src/components/navigation/AppNavigationBoard.vue [hunk 2]
```diff
@@ -36,11 +36,15 @@
 					<button v-if="board.acl.length === 0"
 						class="icon-shared"
 						style="opacity: 0.3"
+						:title="'Share'"
```
Comment (juliushaertl) [inline src/components/navigation/AppNavigationBoard.vue:39]: Please make those translatable.

**Comment by PackElend** [comment]
> Ideally the ActionsButton component from https://github.com/nextcloud/nextcloud-vue should just show a v-tooltip for single actions

@ishantgupta777 can you do so?

**Comment by ishantgupta777** [comment]
> > Ideally the ActionsButton component from https://github.com/nextcloud/nextcloud-vue should just show a v-tooltip for single actions
> 
> @ishantgupta777 can you do so?

Sure sir, I am doing the changes.

File: src/components/navigation/AppNavigationBoard.vue (hunk 2 above)
Comment (ishantgupta777) [reply src/components/navigation/AppNavigationBoard.vue]: @juliushaertl  sir, I have done the changes, please have a look.

File: src/components/boards/BoardItem.vue [hunk 3]
```diff
@@ -32,11 +32,12 @@
 		<div class="board-list-title-cell">
 			{{ board.title }}
 		</div>
-		<div class="board-list-avatars-cell">
-			<Avatar :user="board.owner.uid" class="board-list-avatar" />
+		<div class="board-list-avatars-cell" :title="''">
+			<Avatar :user="board.owner.uid" :title="board.owner.displayname" class="board-list-avatar" />
```
Comment (juliushaertl) [inline src/components/boards/BoardItem.vue:36]:
```suggestion
		<div class="board-list-avatars-cell" title="">
			<Avatar :user="board.owner.uid" :display-name="board.owner.displayname" class="board-list-avatar" />
```

File: src/components/boards/BoardItem.vue [hunk 4]
```diff
@@ -32,11 +32,12 @@
...
 			<Avatar v-for="user in limitedAcl"
 				:key="user.id"
 				:user="user.participant.uid"
+				:title="user.participant.displayname"
```
Comment (juliushaertl) [inline src/components/boards/BoardItem.vue:40]:
```suggestion
				:display-name="user.participant.displayname"
```

File: src/components/navigation/AppNavigationBoard.vue [hunk 5]
```diff
@@ -36,11 +36,15 @@
 					<button v-if="board.acl.length === 0"
 						class="icon-shared"
 						style="opacity: 0.3"
+						:title="t('deck', 'Share')"
```
Comment (juliushaertl) [inline src/components/navigation/AppNavigationBoard.vue:39]:
```suggestion
						v-tooltip="t('deck', 'Share')"
```

Comment (juliushaertl) [reply src/components/navigation/AppNavigationBoard.vue:39]: Same for the next ones ;)

**Comment by ishantgupta777** [comment]
> See inline suggestions

Done. Please have a look.

**Pull request merged**